CHANGES.md

```diff
@@ -1,12 +1,16 @@
-### v2.0.0 (2019-02-24)
+## dev
```

Comment:
I'm confused by the change `###` to `##`, esp. since https://mirage.io/wiki/packaging contains:
```
A MirageOS library should have
- CHANGES.md: containing a log of user-visible changes in each release. For example consider mirage-tcpip CHANGES.md: it has a markdown ### prefix before each release version and the date in (YYYY-MM-DD) form.
```

Reply:
we don't really standardise on the level headers; the CHANGES format won't be homogenous until we mechanise the change parsing more (dune-release does some atm). I just prefer ## to be fewer characters :-)

Reply:
well, then I'd prefer to keep the level as is in each CHANGES file until we settle/mechanise checking on a certain level.

Reply:
fine... pushed